### DISCUSSION TITLE
Error on data transform when training large dirty data

### DISCUSSION THREAD

**Discussion by ChristianEschen** [Project-MONAI/MONAI#5167] [Q&A]
Hi.

I am training a model on a large dataset of dicom files containing "dirty data".

Somewhere in the training phase I receive an error about wrong number of channels.

![Screenshot from 2022-09-16 10-54-18](https://user-images.githubusercontent.com/19329210/190598749-0a688d7b-ba7d-4c1b-8f80-cf7726e5308b.png)

Since I have 300 000 images I cannot make any manual quality assesment of the images.

is there any effective way to create a debug report of images (with paths or more informaton) causing errors?

best regards
Christian

**Comment by wyli** [comment]
thanks for reporting this, there are some discussions about extending the DataAnalyzer module to go through the datasets and perhaps detect this type of input image issue https://github.com/Project-MONAI/MONAI/issues/5168


(currently it can only report image level statistics:
https://github.com/Project-MONAI/MONAI/blob/70c0443a703a8e5f9930d7b677d14425378b9226/monai/apps/auto3dseg/data_analyzer.py#L88-L101)
